fastlane/Fastfile

```diff
@@ -757,6 +760,38 @@ platform :ios do
     macos_build(changelog)
   end
 
+  desc "Clones or updates snapshots repo"
+  lane :fetch_snapshots do
+    Dir.chdir("../Tests") do
+      if !File.directory?(snapshots_repo_name)
+        UI.important("Cloning snapshots repo")
+
+        sh "git", "clone", "git@github.com:RevenueCat/#{snapshots_repo_name}.git"
+      end
+
+      commit = current_snapshots_commit
+      UI.message("Updating snapshots repo to #{commit}")
+      Dir.chdir(snapshots_repo_name) do
+        sh "git", "fetch", "origin"
+        sh "git", "checkout", commit
+      end
```

Comment:
Do we need to check if the repo is dirty at error out before this? 🤔 

Or do we need to do a hard reset so its clean?

Reply:
Good point that would make it more resilient.

Reply:
Actually I'm gonna leave it like this. If it's dirty then you want to handle that error so we don't make you lose whatever local changes you have.